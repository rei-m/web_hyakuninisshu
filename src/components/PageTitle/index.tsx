import * as React from 'react';
import styled from 'styled-components';
import { withAppTheme } from '@src/styles';

export interface PageTitleProps {
  readonly title: string;
}

const H2 = withAppTheme(styled.h1)`
  padding: ${({ theme }) => theme.spacing2x};
  font-size: 2rem;
  margin: 0;
  line-height: 2rem;
`;

const PageTitle: React.SFC<PageTitleProps> = ({ title }) => <H2>{title}</H2>;

export default PageTitle;
