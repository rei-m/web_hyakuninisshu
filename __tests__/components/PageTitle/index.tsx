import * as ReactTestRenderer from 'react-test-renderer';
import * as React from 'react';
import PageTitle from '../../../src/components/PageTitle';

describe('<PageTitle />', () => {
  it('should render component', () => {
    const renderer = ReactTestRenderer.create(<PageTitle title="hoge" />);
    expect(renderer.toJSON()).toMatchSnapshot();
  });
});
