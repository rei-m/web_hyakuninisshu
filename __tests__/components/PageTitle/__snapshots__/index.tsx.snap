// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<PageTitle /> should render component 1`] = `
<h1
  className="sc-bdVaJa fXsTJn"
>
  hoge
</h1>
`;
